CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## 1.0.7
```

Comment:
For the sake of process, I'd put the version number as `## x.x.x` for the commit specific to this feature, and I'll manually bump version numbers before I push. I'll probably walk you thru pushing the plugin at some point as well.